metis-enrichment/metis-enrichment-rest/.gitignore

```diff
@@ -1,2 +1,5 @@
 ##Add to ignore to not commit by mistake
-/src/main/resources/enrichment.properties
\ No newline at end of file
+/src/main/resources/enrichment.properties
+
+# properties file
+**/entity-client.properties
```

Comment:
this can be added to metis-enrichment/.gitignore it will match in modules as well

Reply:
done